### ISSUE TITLE
Run iOS and android tests in parallel

### ISSUE THREAD

**Issue by mmcc007** [mmcc007/screenshots#99]
It should be possible to run iOS and android integration tests in parallel. This can be done by separating out the tests by device type (iOS or android) and running in a separate isolate for each device type (iOS or android).

**Comment by patreu22** [comment]
As far as I remember fastlane's snapshot also supports simultanous iOS simulators, that would also great to follow!

**Comment by mmcc007** [comment]
Yes, since iOS simulators can run in parallel out of the box, it should be possible to run the iOS simulators in a pool of isolates for faster results.

AFAIK, android emulators do not yet run in parallel out-of-the-box. So for now, android runs would be in series.
